runtime/doc/news.txt

```diff
@@ -36,6 +36,12 @@ OPTIONS
 • 'jumpoptions' flag "unload" has been renamed to "clean".
 • The `msghistory` option has been removed in favor of 'messagesopt'.
 
+TREESITTER
+
+• *TSNode:child_containing_descendant()* has been removed in the tree-sitter
```

Comment:
Nit:

```suggestion
• *TSNode:child_containing_descendant()* has been removed in the Treesitter
```

I believe that no dash is the convention: https://github.com/neovim/neovim/pull/27110

Reply:
Ah, "treesitter" is what we call the functionality it in our codebase, but the upstream project is still called "tree-sitter". So this is me making clear who's to blame here ;)

Reply:
Well FWIW the rest of this file uses "Treesitter", so let's at least try to aim for file consistency 😅

Reply:
Yes but the rest of the file is not literally talking about the upstream project.

Reply:
Oh I see. I think that the use of "Treesitter library" already makes that clear, but I'll let you decide.

Reply:
(This is temporary in any case, since it's a HEAD-only change so won't survive the next release.)